### ISSUE TITLE
Missing from firefox addon store?

### ISSUE THREAD

**Issue by Brayconn** [JohannesFischer/soundcloud-control#16]
The addon has been removed from the firefox addon store as of a few weeks(?) ago.

Steps to reproduce:
1. Visit https://addons.mozilla.org/en-US/firefox/addon/soundcloud-control-extension/
2. ???
3. Profit

**Comment by JohannesFischer** [comment]
It as removed with the release of Firefox 69 and the guidelines for extensions changed. The issue is that the bundled extension contains minified code and is therefore being rejected in the review process.
I didn't get around making the changes need but hope that someone will pick his up during hacktober :smile:

**Comment by Tobsic** [comment]
I just asked in the IRC Channel, if it's ok to use Webpack without minification. If so, you just need to set `optimization.minimizer` to `false` in the `webpack.conf.js`. Would be awesome to get the addon on my new machine. Thanks for this great extension :)

**Comment by kewisch** [comment]
Hi @JohannesFischer . Chiming in from the Firefox review team here. You are also welcome to continue to use minification, just not obfuscation. From a brief look I didn't see any obfuscation, so there may have been a misunderstanding. See https://extensionworkshop.com/documentation/publish/source-code-submission/#use-of-obfuscated-code for details. I've also replied in the reviewer platform, if you could provide sources as per said link I'm sure we can get your add-on up and running again soon. Thanks @Tobsic for bringing this up!

**Comment by JohannesFischer** [comment]
Brilliant, thank you so much for looking into this @Tobsic I had a lot going on and didn't have time to look into this. But if only a config change is enough the extension should be back very soon, so you and can use it again :)
Also thank you @kewisch for the clarification. I somehow mixed the terms up when replying to this issue.

**Comment by chidi9901** [comment]
Any updates on that? It's still not online.

**Comment by JohannesFischer** [comment]
@chidi9901 Nothing so far but I have some time off next week. Will see that I get the addon listed in the store again.

**Comment by JohannesFischer** [comment]
I just submitted a new version of the add-on for review. Some of the changes made include replacing Webpack with Rollup, just to make the build process simpler. Webpack always felt like overkill just to get `import` working in such a simple extension. I also removed all external dependencies since they are no longer need to set custom keyboard shortcuts.

It shouldn't be long to hear back from the reviewer and to hopefully have the add-on back in the store.
